documentation/docs/client/reactions.md

```diff
@@ -0,0 +1,110 @@
+---
+id: clientReactions
+title: Reactions
+sidebar_position: 18
+---
+
+Stream Chat has built-in support for user Reactions. Common examples are likes, comments, loves, etc. Reactions can be customized so that you are able to use any type of reaction your application requires.
```

Comment:
```suggestion
Stream Chat has built-in support for user reactions. Common examples are likes, comments, loves, etc. Reactions can be customized so that you are able to use any type of reaction your application requires.
```